google-cloud-firestore/src/test/java/com/google/cloud/firestore/it/ITSystemTest.java

```diff
@@ -368,13 +368,107 @@ public void defaultQuery() throws Exception {
     assertEquals("bar", documents.next().get("foo"));
   }
 
+  @Test
+  public void defaultQueryStream() throws Exception {
+    addDocument("foo", "bar");
```

Comment:
I would prefer having different field values to make sure each DocumentSnapshot we are getting is different.